pandas/tests/frame/indexing/test_setitem.py

```diff
@@ -339,3 +339,17 @@ def test_setitem_boolean_mask(self, mask_type, float_frame):
         expected = df.copy()
         expected.values[np.array(mask)] = np.nan
         tm.assert_frame_equal(result, expected)
+
+    @pytest.mark.parametrize("func", [list, np.array, Series])
+    @pytest.mark.parametrize("value", [[], [False]])
```

Comment:
it isnt clear to me that this is correct with `[]`.  why is it included?

Reply:
Shouldn't an empty indexer do nothing to the underlying object?